assets/css/pages/home/how_we_do_it.css

```
@@ -0,0 +1,85 @@
+
+/* How we do it */
+.step-name {
+    font-size: 36px;
+    color: var(--brand-pink);
+}
+
+.step-text {
+    align-self: center;
+    color: #332859;
+    font-size: 25px;
+    line-height: 2em;
+}
+
+.step-image img {
+    width: 350px;
+    margin: 0 20px;
+}
+
+.step-item {
+    display: flex;
+    margin: 0px 200px;
+    margin-bottom: 100px;
+    font-weight: 100;
+    align-items: center;
+    align-self: center;
+}
+
+.half-margin-bottom {
+    margin-bottom: 50px;
+}
+
+@media only screen and (max-width: 1100px) {
+
+    .step-item {
+        margin: 30px 0 30px 30px;
+    }
+
+    .step-image img {
+        width: 300px;
+    }
+}
+
+@media only screen and (max-width: 768px) {
+
+    .step-item {
+        flex-direction: column;
+        padding: 0;
+        align-items: start;
+        margin: 0 30px !important;
+    }
+
+    .reverse-column {
+        flex-direction: column-reverse;
+    }
+
+    .step-name {
+        font-size: 30px;
+        text-align: center;
+    }
+
+    .step-text {
+        font-size: 25px;
+        line-height: 1.5em;
+    }
+
+    .step-image {
+        align-self: center;
+    }
+
+    .step-image img {
+        padding: 5px 10px 20px 10px;
+        max-width: 100vw;
+        width: 200px !important;
+        margin: 20px auto;
+    }
+
+    .pink-stripe {
+        margin-top: 50px;
+    }
+
+    .blue-stripe {
+        margin-bottom: 50px;
+    }
+}
```

another missing line
```suggestion
}

```